scripts/Test.ps1

```diff
@@ -115,9 +117,14 @@ $testProjects | ForEach-Object {
     $exitCode = $LASTEXITCODE # Save exit code in case upload results fails
 
     if($ResultsDirectory -and $Logger -and $UploadResultsToAppVeyor -and $env:APPVEYOR_JOB_ID) {
-        $resultsLog = Join-Path -Path $ResultsDirectory -ChildPath $logFileName
-        if(Test-Path -Path $resultsLog) {
+		Write-Host "Publishing test results..."
+		#Write-Host "ResultsDirectory: $ResultsDirectory"
+		#Write-Host "LogFileName: $logFileName"
```

Comment:
This section of code is doubly indented. I would recommend removing these commented out console logs if you feel they aren't needed anymore.